src/utils.rs

```diff
@@ -86,14 +86,14 @@ pub struct CallGraph {
 #[allow(clippy::missing_panics_doc, clippy::expect_used, clippy::unwrap_used)]
 pub fn configure_rustup_toolchain() {
     let rsup_default = Command::new("rustup")
-        .arg("default")
+        .args(["show", "active-toolchain"])
         .output()
         .expect("failed to run rustup to configure toolchain");
     let utf8_toolchain = std::str::from_utf8(&rsup_default.stdout)
```

Comment:
We now use `rustup show active-toolchain` to determine the toolchain to use. This should be *slightly* more reliable than `rustup default`, in the sense that it'll both respect user overrides and should (?) respect the local `rust-toolchain.toml` in any crate.